dallinger/command_line.py

```diff
@@ -334,7 +334,18 @@ def debug(verbose, bot, exp_config=None):
             # Call endpoint to launch the experiment
             log("Launching the experiment...")
             time.sleep(4)
-            requests.post('{}/launch'.format(base_url))
+            launch_request = requests.post('{}/launch'.format(base_url))
+            try:
+                launch_data = launch_request.json()
+            except ValueError:
+                error("Error parsing response from /launch: {}".format(launch_request.content))
+                raise
+
+            if not launch_request.ok:
+                error('Experiment launch failed, check web dyno logs for details.')
```

Comment:
If the launch route errors out in an unexpected way then this message won't be shown, will it? It looks like it'd show if /launch returns valid JSON but the status code isn't 200. Do you think we could move the raise lower, and have line 346 gracefully fail if there's no JSON?